tcms/core/tests/test_admin.py

```diff
@@ -44,21 +43,22 @@ def test_admin_display(self):
 
         self.assertNotContains(response, 'Test case status')
         self.assertNotContains(response, 'Test case texts')
-        self.assertNotContains(response, 'Test cases')
+        self.assertContains(response, 'Search Test Cases')
 
         # for tcms.testplans
         self.assertContains(response, 'Plan types')
 
-        self.assertNotContains(response, 'Test plans')
+        self.assertContains(response, 'Search Test Plans')
 
         # for tcms.testruns
-        self.assertNotContains(response, 'Testruns')
-        self.assertNotContains(response, 'Test case run statuss')
+        self.assertNotContains(response, 'Test case run status')
         self.assertNotContains(response, 'Test case run statuses')
-        self.assertNotContains(response, 'Test runs')
+        self.assertContains(response, 'Search Test Runs')
 
-        # for tcms.xmlrpc
-        self.assertContains(response, 'Xml rpc logs')
+        # for tcms main modules
+        self.assertContains(response, 'Testruns')
+        self.assertContains(response, 'Testcases')
+        self.assertContains(response, 'Testplans')
```

Comment:
these must be groupped with their corresponding sections above otherwise makes the asserts hard to read